drizzlepac/devutils/comparison_tools/interdetector_sourcelist_crossmatch.py

```diff
@@ -150,11 +149,11 @@ def run(sl_names, img_names, diagnostic_mode=False, log_level=logutil.logging.IN
     for diff_ra, title, units in zip(diff_list, title_list, units_list):
         log.info("Comparison - reference {} statistics ({})".format(title, units))
 
-        compute_stats(diff_ra, title)
+        foo = compute_stats(diff_ra, title)
```

Comment:
Since the variable `foo` does not seem to be used anywhere else in this routine, please replace with the Pythonic `_`.